TicTacToe/Checker.cpp

```diff
@@ -45,4 +45,20 @@ bool Checker::winner() {
 		return true;
 	else 
 		return false;
+}
+
+void Checker::displayBoard() {
+	for (int row = 0; row < 9; row += 3) {
+		for (int col = row; col < row + 3; col++) {
+			std::cout << "[" << board_[col] << "]";
+		}
+		std::cout << std::endl;
+	}
+}
+
+bool Checker::sameThree(int one, int two, int three) {
+	if (one == two && two == three && one == three) {
+		return true;
```

Comment:
it will be the same if you simply checks like 
```
if (one == two == three) {
}
```

because (one == two) -> will be true or false. and ((one == two) == three)